model/project_ref.go

```diff
@@ -1950,6 +1953,12 @@ func SaveProjectPageForSection(projectId string, p *ProjectRef, section ProjectP
 			bson.M{
 				"$set": bson.M{projectRefPeriodicBuildsKey: p.PeriodicBuilds},
 			})
+	case ProjectPageContainerSection:
+		err = db.Update(coll,
```

Comment:
Does this need to validate the container sizes before updating them?

Reply:
There should be form validation on Spruce's side to ensure that memory and CPU are properly defined, but we can add validation here as well for additional assuredness.

Reply:
I'll add it for some other sections too while I'm at it since they don't have validation here either.